javaslang/src/main/java/javaslang/Value.java

```diff
@@ -818,6 +819,39 @@ default CharSeq toCharSeq() {
      */
     @Override
     String toString();
+    
+    
+    /**
+     * Collect value using provided collector
+     * <p>
+     * Use {@link #collect(Supplier, BiConsumer, BiConsumer)} internally 
+     * 
+     * @param collector Collector performing reduction
+     * @return R result of "reduction" with collector
+     */
+    default <R, A> R collect(Collector<? super T,A,R> collector) {
+		return collector.finisher()
+				.apply(this.collect(collector.supplier(), 
+						collector.accumulator(), null));
+    }
+
+    /**
+     * Collect value using supplier, accumulator and combiner
+     * <p>
+     * Since it is provided for a single value, it do not use combiner
```

Comment:
@earlzero yes, that's right, we need the `combiner`, too. Could you please add it?